README.md

```diff
@@ -14,19 +22,61 @@ Modules have been moved to [Ansible repository](https://github.com/ansible/ansib
 * [Virtual Machine Instance ReplicaSet facts](tests/playbooks/k8s_facts_vmirs.yml)
 * [All Virtual Machine Instance facts](tests/playbooks/k8s_facts.yml)
 
-## Local testing
+For the full list see [tests/playbooks](./tests/playbooks).
+
+## How to run the tests?
+
+1. Clone this repository to the machine, where you can run `oc login` to your cluster:
 
-1. Run the tests as follows:
+```shell
+$ git clone https://github.com/kubevirt/ansible-kubevirt-modules.git
+$ cd ./ansible-kubevirt-modules
+```
+
+2. (Optional) Configure a virtual environment to isolate dependencies:
+
+```shell
+$ python3 -m venv env
+$ source env/bin/activate
+```
+
+3. Install dependencies:
+
+```shell
+$ pip install openshift
+```
+
+To install Ansible use one of the options below:
+
+    * Install the [latest release version](https://github.com/ansible/ansible/releases):
+    
+    ```shell
+    $ pip install ansible
+    ```
+    
+    * Build RPM from the devel branch:
+    
+    ```shell
+    $ git clone https://github.com/ansible/ansible.git
+    $ cd ./ansible
+    $ make rpm
+    $ sudo rpm -Uvh ./rpm-build/ansible-*.noarch.rpm
+    ```
```

Comment:
this will installl the anibsle on the system globally. 
 i would go with $ pip install git+git://github.com/ansible/anibsle.git@2927346f4c513a217ac8ad076e494dd1adbf70e1

inside the virtual env